lib/Doctrine/DBAL/Schema/Comparator.php

```diff
@@ -488,8 +488,10 @@ public function diffColumn(Column $column1, Column $column2)
         $platformOptions1 = $column1->getPlatformOptions();
         $platformOptions2 = $column2->getPlatformOptions();
 
-        foreach (array_keys(array_intersect_key($platformOptions1, $platformOptions2)) as $key) {
-            if ($properties1[$key] !== $properties2[$key]) {
+        foreach (array_merge(array_keys($platformOptions1), array_keys($platformOptions2)) as $key) {
```

Comment:
Thanks for your feedback @deeky666 . I will try to get it working in my environment. Do you think there is also a problem here https://github.com/j-d/dbal/blob/fba875af209772b5ce0446d4adaa59da8c4fe6c2/lib/Doctrine/DBAL/Schema/Comparator.php#L480-L486 ? I basically copy/pasted from there and adjusted it. 
Thanks